components/spin/index.tsx

```diff
@@ -25,6 +25,8 @@ export interface SpinProps {
   wrapperClassName?: string;
   indicator?: SpinIndicator;
   children?: React.ReactNode;
+  fullscreen?: boolean;
+  onClick?: (e: React.MouseEvent<HTMLDivElement>) => void;
```

Comment:
I don't think we need `onClick` event

Reply:
I kinda agree, I just added to match the material-ui approach, I will remove and change the way that demo works!

Reply:
It can be kept temporarily. Regarding onClick, this still needs to be discussed by the antd team.

Reply:
@MadCcc @zombieJ Do you think `onClick` should be retained?

Reply:
All right!, I will add to the `onClick` to the docs if gonna kept it.

Reply:
In my view, Spin should not provide `onClick` since we do not expect user to interactive with this.